### PR TITLE
[NEXT-21477] check deliveries for length

### PR DESCRIPTION
**Pull request by pakeil**
### 1. Why is this change necessary?
Possible fix for 
https://issues.shopware.com/issues/NEXT-21477

### 2. What does this change do, exactly?
Check the array for length

### 3. Describe each step to reproduce the issue or behaviour.
See issue tracker

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Core/Content/ImportExport/DataAbstractionLayer/Serializer/Entity/OrderSerializer.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function serialize(Config $config, EntityDefinition $definition, $entity)
             $entity['lineItems'] = implode('|', $modifiedLineItems);
         }
 
-        if (isset($entity['deliveries'])) {
+        if (isset($entity['deliveries']) && count($entity['deliveries']) > 0) {
```
Comment (shyim) [inline src/Core/Content/ImportExport/DataAbstractionLayer/Serializer/Entity/OrderSerializer.php:44]: That looks like a broken order when it doesn't have any deliveries

Comment (pakeil) [reply src/Core/Content/ImportExport/DataAbstractionLayer/Serializer/Entity/OrderSerializer.php:44]: This addresses orders with just one line item which is a "credit". If these orders are supposed to have deliveries, this change can be rejected/closed.

**Comment by shyim** [comment]
@pakeil Can you accept the CLA?

**Comment by pakeil** [comment]
sla signed

**Comment by keulinho** [comment]
@pakeil thanks again for your contribution, your changes were merged! :blue_heart: :tada:

**Pull request closed without merging**